src/bridge/handlers/authHandler.ts

```
@@ -0,0 +1,53 @@
+      setAccessToken(accessToken);
+    }
+  }
+
+  return {
+    type: 'auth:login',
+    success: true,
+  };
+};
+
+const handleLogout = async (): Promise<AuthResponse> => {
+  const { clearAuth } = useAuthStore();
```

Biome (2.4.4)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)